src/software/field_tests/movement_robot_field_test.py

```diff
@@ -76,144 +75,144 @@
-    if len(world.friendly_team.team_robots) == 0:
-        raise Exception("The first world received had no robots in it!")
-
-    print("Here are the robots:")
-    print(
-        [
-            robot.current_state.global_position
-            for robot in world.friendly_team.team_robots
-        ]
-    )
-
-    id = world.friendly_team.team_robots[0].id
-    print(f"Running test on robot {id}")
-
-    point1 = Point(x_meters=-0.3, y_meters=0.6)
-    point2 = Point(x_meters=-0.3, y_meters=-0.6)
-    point3 = Point(x_meters=-1.5, y_meters=-0.6)
-    point4 = Point(x_meters=-1.5, y_meters=0.6)
-
-    tactic_0 = MoveTactic(
-        destination=point1,
-        final_speed=0.0,
-        dribbler_mode=DribblerMode.OFF,
-        final_orientation=Angle(radians=-math.pi / 2),
-        ball_collision_type=BallCollisionType.AVOID,
-        auto_chip_or_kick=AutoChipOrKick(autokick_speed_m_per_s=0.0),
-        max_allowed_speed_mode=MaxAllowedSpeedMode.PHYSICAL_LIMIT,
-        target_spin_rev_per_s=0.0,
-    )
-    tactic_1 = MoveTactic(
-        destination=point2,
-        final_speed=0.0,
-        dribbler_mode=DribblerMode.OFF,
-        final_orientation=Angle(radians=-math.pi / 2),
-        ball_collision_type=BallCollisionType.AVOID,
-        auto_chip_or_kick=AutoChipOrKick(autokick_speed_m_per_s=0.0),
-        max_allowed_speed_mode=MaxAllowedSpeedMode.PHYSICAL_LIMIT,
-        target_spin_rev_per_s=0.0,
-    )
-    tactic_2 = MoveTactic(
-        destination=point3,
-        final_speed=0.0,
-        dribbler_mode=DribblerMode.OFF,
-        final_orientation=Angle(radians=-math.pi / 2),
-        ball_collision_type=BallCollisionType.AVOID,
-        auto_chip_or_kick=AutoChipOrKick(autokick_speed_m_per_s=0.0),
-        max_allowed_speed_mode=MaxAllowedSpeedMode.PHYSICAL_LIMIT,
-        target_spin_rev_per_s=0.0,
-    )
-    tactic_3 = MoveTactic(
-        destination=point4,
-        final_speed=0.0,
-        dribbler_mode=DribblerMode.OFF,
-        final_orientation=Angle(radians=-math.pi / 2),
-        ball_collision_type=BallCollisionType.AVOID,
-        auto_chip_or_kick=AutoChipOrKick(autokick_speed_m_per_s=0.0),
-        max_allowed_speed_mode=MaxAllowedSpeedMode.PHYSICAL_LIMIT,
-        target_spin_rev_per_s=0.0,
-    )
-    tactics = [tactic_0, tactic_1, tactic_2, tactic_3]
-
-    for tactic in tactics:
-        print(f"Going to {tactic.destination}")
-        params = AssignedTacticPlayControlParams()
-        params.assigned_tactics[id].move.CopyFrom(tactic)
-
-        field_test_runner.set_tactics(params, True)
-        field_test_runner.run_test(
-            always_validation_sequence_set=[[]],
-            eventually_validation_sequence_set=[[]],
-            test_timeout_s=4,
-        )
-
-    # Send a stop tactic after the test finishes
-    stop_tactic = StopTactic()
-    params = AssignedTacticPlayControlParams()
-    params.assigned_tactics[id].stop.CopyFrom(stop_tactic)
+# def test_basic_rotation(field_test_runner):
```

Comment:
is this intentional?

Reply:
For some reason these tests are failing locally on my end when I run `bazel test //software/...:all` so not too sure, I forgot to ask about it last Saturday.